Plugin.php

```diff
@@ -65,6 +66,9 @@ protected function applyBackendSkin()
             Config::set('brand.logoPath', '~/modules/backend/assets/images/winter-logo-white.svg');
             // Config::set('brand.logoPath', '~/modules/backend/assets/images/winter-logo.svg');
         }
+
+        $pluginUrl = url(Config::get('cms.pluginsPath') . '/winter/tailwindui');
```

Comment:
This would need to be the `Url::asset()` helper instead to support assets being published to a CDN instead of locally hosted on the web server.

Reply:
Thanks. Will adjust it.

Reply:
What I found out is that append to the 'head' block does not work when on the backend dashboard page. So currently the dashboard page does not toggle dark mode.